Lesson Plans/Private/XC.9 Emergency Operations.md

```diff
@@ -23,42 +23,107 @@ Develop knowledge of all abnormal and emergency procedures for the training airc
+	- Airspeed isn't "live"
+		- Abort takeoff, or fly the pattern and land normally
+	- A door opens
+		- Takeoff normally.  Deal with it in the air, or land normally
+- In the air...
+	- Fire!  Where?
+		- Engine cowling?
+			- **(POH)** Cut off fuel flow; Emergency Descent Procedure; don't turn off electrical unless fire is electrical
+		- Inside the dashboard (probably electrical) or cabin?
+			- **(POH)** Master and stby switches off; deploy fire extinguisher
+				- Fire extinguished? - open windows to clear smoke; if required, follow procedure to re-power some items; Precautionary Landing
+				- Not extinguished? - Emergency Descent
+		- Wing?
+			- **(POH)** Turn wing lights and pitot heat off; sideslip to keep flames away; Precautionary landing
+	- Burning smell! (Possible electrical fire)
+		- **(POH)** Master and stby switches off; optionally follow procedure to re-power some items
+	- Engine failure!
+		- **(POH)** Best glide; aim for landing spot; attempt restart; Else execute Forced Landing
+	- Engine running poorly!
+		- Identify a landing spot and navigate toward it; Climb if possible, to buy time; Debug the problem, but be prepared for a Forced Landing
+		- Possible problems include:
+			- Carburetor icing - Add carb heat. [[How and Why Carburetor Ice Forms]]
+			- Bad mixture/throttle settings - Adjust
+			- Fuel pump failure - Turn on backup pump
+			- Fuel tank running dry - Switch tanks
+			- Ignition switch in 'A' or 'B' - Switch to 'both'
+	- Electrical failure
+		- The plane still flies; use backup radio; land ASAP
+	- Radio failure (you can't hear them, or they can't hear you)
+		- Squawk 7600; proceed to destination if safe; look for light signals
+	- Instruments don't seem to be reading correctly
+		- Are altimeter and VSI "stuck" even when you change altitude?
+			- **(POH) **Static source is blocked; try alternate static source
+		- Does airspeed vary with altitude, even when flying level?
+			- Pitot tube is blocked
+		- Are the vacuum-powered instruments dead? (Which ones are they in your plane?)
+			- Vacuum pump is dead
+	- Plane isn't handling correctly:
+		- Debug if control surfaces have failed, or flaps aren't extending, or one is stuck
+		- Learn to control the plane with what you have; don't forget trim can help; make a precautionary landing
+	- A door opens - Deal with it in the air, or land normally
+	- Low oil pressure - Could lead to engine failure.  Precautionary landing
+	- Low fuel - Divert to alternate to refuel
+	- Low battery - Shed as much load as possible; estimate time available; land accordingly
+	- Retractable gear malfunction - (out of scope for this lesson)
+	- Flew into a cloud, or lost minimum VFR visibility
+		- Rely on [[XC.8 Emergency Flight by Instruments - Four Fundamentals]] skills; reverse course; request help on the radio
+	- Icing - Reverse course or choose a different altitude; request help on the radio
```

Comment:
C172SP POH actually has several things to do here, like leave wing flaps up for landing, open windows and scrap of ice from windshield, and others

Reply:
Good points.  I added some wording.